dashboard/azimuth_plot.py

```diff
@@ -0,0 +1,88 @@
+'''Produces the skyplot for a given region at a given time'''
+#pylint: disable=E0401,C0413,W0612,E1101
+import sys
+import os
+from datetime import datetime as dt
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
+from dotenv import load_dotenv
+sys.path.append(os.path.abspath(os.path.join(
+    os.path.dirname(__file__), '..', 'weekly-openmeteo')))
+from extract import get_connection
+
+
+def get_azimuth_data(region: str, time_point:dt):
+    '''Queries the database for azimuth data'''
+    query = '''SELECT b.body_name, ba.azimuth, ba.altitude, ba.distance_km FROM region as r
+    JOIN body_assignment as ba USING (region_id)
+    JOIN body as b USING (body_id)
+    WHERE r.region_name = %s AND
+    ba.at = %s'''
+    with get_connection() as conn:
+        with conn.cursor() as cur:
+            cur.execute(query,(region,time_point))
+            data = cur.fetchall()
+
+    return data
+
+
+def transform_azimuth_data(data: list) -> pd.DataFrame:
```

Comment:
Good understanding of how to generate skyplots, using radians as opposed to the default measure of azimuth.